sdk/src/main/scala/com/horizen/account/api/rpc/service/EthService.scala

```diff
@@ -589,4 +589,107 @@ class EthService(
         .toArray
     }
   }
+
+  @RpcMethod("eth_feeHistory")
+  @RpcOptionalParameters(1)
+  def feeHistory(
+      blockCount: Quantity,
+      newestBlock: String,
+      rewardPercentiles: Array[Double]
+  ): EthereumFeeHistoryView = {
+    val percentiles = sanitizePercentiles(rewardPercentiles)
+    applyOnAccountView { nodeView =>
+      val genesisHeight = nodeView.history.getBlockHeightById(networkParams.sidechainGenesisBlockId).get()
```

Comment:
genesis height is always `1` in our case. We may simplify

Reply:
done